src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Mocks/IUnconfiguredProjectCommonServicesFactory.cs

```diff
@@ -13,7 +13,9 @@ public static IUnconfiguredProjectCommonServices Create()
             return mock.Object;
         }
 
-        public static IUnconfiguredProjectCommonServices Create(UnconfiguredProject project = null, IPhysicalProjectTree projectTree = null, IProjectThreadingService threadingService = null, ConfiguredProject configuredProject = null, ProjectProperties projectProperties = null)
+        public static IUnconfiguredProjectCommonServices Create(UnconfiguredProject project = null, IPhysicalProjectTree projectTree = null, IProjectThreadingService threadingService = null, 
+                                                                ConfiguredProject configuredProject = null, ProjectProperties projectProperties = null,
+                                                                IProjectLockService projectLockService = null)
         {
             var mock = new Mock<IUnconfiguredProjectCommonServices>();
```

Comment:
replace with `Mock.Of<IUnconfiguredProjectCommonServices>()` but the current one works too